podvm/Dockerfile.podvm

```diff
@@ -13,12 +13,23 @@ ARG PODVM_DISTRO=ubuntu
 ARG AA_KBC=offline_fs_kbc
 # If not provided, uses system architecture
 ARG ARCH=x86_64
+ARG CAA_SRC=""
+ARG CAA_SRC_REF=""
```

Comment:
You want CAA_SRC CAA_SRC_REF to be available only as build time options right, and hence you don't add it to ENV ?

Reply:
Yest @bpradipt , they are build time options, I didn't see any reason to make them exported to runtime environment.